### PR TITLE
chore(examples): add `axum` example

### PR DESCRIPTION
**Pull request by j3rrywan9**

### CODE REVIEW

File: examples/axum.rs [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+}
+
+pub async fn metrics_handler(State(state): State<Arc<Mutex<AppState>>>) -> impl IntoResponse {
+    let state = state.lock().await;
+    let mut buffer = String::new();
+    encode(&mut buffer, &state.registry).unwrap();
+
+    Response::builder()
+        .status(StatusCode::OK)
+        .header(CONTENT_TYPE, "application/openmetrics-text; version=1.0.0; charset=utf-8")
```
Comment (mxinden) [inline examples/axum.rs:50]: :rocket:

**Review by mxinden — COMMENTED** [review]
Neat! Thank you.

Can you address the CI failures? Otherwise, this looks good to me.

**Comment by j3rrywan9** [comment]
@mxinden Thanks for the review! I've fixed all the issues called out by `cargo fmt`

**Review by mxinden — APPROVED** [review]
Thank you!

**Pull request merged**